source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -178,6 +201,7 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
         selectedDate = datePicker.dateValue
         datePickerCalendar.dateValue = datePicker.dateValue
         datePickerTextfield.dateValue = datePicker.dateValue
+        clearButton.isHidden = false
     }
```

Comment:
Guess this is not required

Reply:
Akaash without this
The clear button won't reappear once date is picked again without this